### PR TITLE
Step time biofeedback (StepTB)

### PR DESCRIPTION
**Pull request by jsvoo**
Incuded StepTB about information on the Portfolio page and added StepTB portfolio page according to issue #112

StepTB About page Update
image

StepTB Portfolio Page
image

### CODE REVIEW

File: docs/project_step_time_biofeedback/100x100.jpg
Comment (kungfuchicken) [inline docs/project_step_time_biofeedback/100x100.jpg:1]: This is the logo for Seeing Is Believing. We can remove any logo until we identify a new logo, but we probably shouldn't re-use that one here.

File: docs/project_step_time_biofeedback/about.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- **Source Code:** [https://github.com/oss-slu/Step_Time_Biofeedback](https://github.com/oss-slu/Step_Time_Biofeedback) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/Step_Time_Biofeedback)
+- **Client:** 
+  - [Dr. Jason Longhurst, Ph.D., PT, DPT, NCS](https://www.slu.edu/doisy/faculty/longhurst-jason.php)
+- **Current Tech Lead:** Victor Ojogbane Onoja [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/jsvoo)
+- **Developers:**
+
+  - TBD
+
+- **Start Date:** August 2024
+- **Adoption Date:** TBD
```
Comment (kungfuchicken) [inline docs/project_step_time_biofeedback/about.md:23]: Adoption date would be same as start date, since this is a greenfield project.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
almost there!

**Comment by jsvoo** [comment]
@kungfuchicken Both changes have been implemented.

**Review by kungfuchicken — APPROVED** [review]
lgtm;

**Pull request merged**